jdi-light-material-ui-tests/src/main/java/io/github/com/pages/surfaces/AppBarFrame.java

```diff
@@ -0,0 +1,16 @@
+package io.github.com.pages.surfaces;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.material.elements.surfaces.AppBar;
+
+public class AppBarFrame extends Section {
+    @UI(".MuiIconButton-label")
+    public AppBar menu;
+
+    @UI("//h6")
```

Comment:
better change locator to .MuiTypography-root
because title size can be changed in feature to h5 for example